frontstage/views/passwords/reset_password.py

```diff
@@ -119,27 +119,20 @@ def request_password_change(email):
     party_id = str(respondent["id"])
     password_reset_counter = party_controller.get_password_reset_counter(party_id)["counter"]
 
-    # When the password verification token has expired, it is deleted from the DB. The password counter would always be
-    # >0 and the the try below would be entered (when only the passworrd_reset_counter was in the if). When this
-    # occurred, the code would error as the password_verfication_token would be missing from the dictionary. I've added
-    # a check to ensure that this never enters the try when this is missing from the dict.
-    if password_reset_counter != 0 and "password_verification_token" in respondent:
+    # When the password_verification_token has expired, it will be deleted from the DB
```

Comment:
I'm questioning this statement, I don't there is anything that deletes it like this. I've however checked with and without and it is fine